src/main/java/Interpreter.java

```diff
@@ -0,0 +1,23 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Interpreter {
+
+
+    public List<String> getOperands(String inputExpression) {
```

Comment:
Wouldnt it be easier if that returns `List<Integer>` ?

Reply:
i'll think about it